docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: nexus
+title: Temporal Nexus - Temporal feature
+description: Explore how Temporal Nexus enables modular architecture and secure multi-team collaboration by abstracting Temporal primitives into clean Nexus Service APIs across namespace boundaries.
+sidebar_label: Temporal Nexus
+slug: /temporal-nexus
+tags:
+  - temporal-nexus
+  - temporal-cloud
+
+keywords:
+  - temporal multi-team collaboration
+  - modular temporal architecture
+  - namespace boundary workflows
+  - temporal nexus api registry
+  - temporal nexus services
+  - secure temporal workflows
+  - multi-team integration temporal
+  - temporal cloud nexus
+---
+
+:::tip Support, stability, and dependency info
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for both OSS and Temporal Cloud.
+
+:::
+
+
+Temporal Nexus is an integrated feature of the Temporal platform designed to connect durable executions across team and Namespace boundaries.
+
+When organizations build applications, whether small or large, it’s essential to design them in a modular way to ensure scalability and adaptability. 
```

Comment:
```suggestion
Temporal keeps the runtime benefits of an Event-Driven Architecture (EDA) where services can be down without affecting each other, but offers a higher-level durable execution programming model that lets you write idiomatic code in any language and preserve the state of execution so it can be resumed automatically in the presence of failure, without the extensive boilerplate code and complexity that comes with using raw EDA frameworks directly.
```